Dockerfile

```diff
@@ -10,14 +10,9 @@ ENV CGO_ENABLED=0
 RUN go build -a -mod vendor -o exo \
         -ldflags "-s -w -X main.version=${VERSION} -X main.commit=${VCS_REF}"
 
-FROM alpine:3.12.0 as ca-certificates
-
-RUN apk add --no-cache ca-certificates
-
-FROM scratch
+FROM alpine:3.12.0
```

Comment:
Is it possible to replace with `3.12` to automatically pull the latest bugfix of `3.12` thanks :)